contents/docs/data/cohorts.mdx

```diff
@@ -22,97 +22,83 @@ availability:
 
-![click data point on trend to show persons](../../images/docs/user-guides/funnels/trend-click-on-data-point.png)
+**Step 1:** Go to [insights](https://app.posthog.com/insights) and open any insight. Click on any data point to view the [persons](/docs/data/persons) represented in the underlying data.
 
 **Step 2:** In the modal popup, click 'Save as cohort' in the bottom right.
 
-![create cohort from trend modal](../../images/features/cohorts/create-cohort-from-trend.png)
-
-## Cohort types
-
-When creating a cohort from scratch, you'll first choose between two type of cohorts, static and dynamic:
-
-![two types of cohorts: static and dynamic](../../images/features/cohorts/static-and-dynamic-cohorts.png)
-
-### Static cohorts
-
-You might want to batch users together based on the value of some mutable property at a certain point in time. So, for example, if you wanted to save a cohort of users you experimented on, and selected them based on usage at a given point in time, you might want to 'freeze' that list and have it accessible to track them at a later point in time. To do so, you can upload a csv of users. Unlike a dynamic cohort, a static cohort will not change as user properties/events change.
+The video below shows how to create a new cohort from a trend, although you can similarly create a cohort from a funnel, user path, or any other insight.
 
-### Dynamic cohorts
+![How to create a cohort from a trend](../../images/features/cohorts/create-cohort-from-trend.mp4)
 
-##### Match users who performed action or event
+### Method 2: Create a new cohort from the cohorts page
 
-Sometimes, you'll want to group users that have performed a certain event or [actions](/docs/user-guides/actions) within a given time. For example, you might want to track 'daily active' or 'weekly active' users, based on the frequency of them performing a given action. For this, you'll want a list that dynamically updates, moving users in or out as they meet criteria or not.
+**Step 1:** Navigate to the [Cohorts](https://app.posthog.com/cohorts) page in the sidebar:
 
-##### Match users who have properties
-
-Or, you might select a user property to filter on, and then update that property, and with a dynamic cohort, the user could be filtered in or out. Commonly, this used to combine users at a certain team, from a certain marketing campaign (via their UTM tags), email, etc.
-
-![Three key cohort settings](../../images/features/cohorts/cohort-options.png)
-
-**'Add matching criteria'** adds another group of filters that a user must match to be a part of the cohort, as an `OR` operation. This means that if a user matches _any_ of the groups you add, they will be added to the cohort.
-
-##### Match users based on their lifcycle
+**Step 2:** Click '+ New Cohort' on the top right.
 
-<FeatureAvailability availability={_frontmatter.availability.features.lifecycle} />
+![left hand navigation for cohorts](../../images/features/cohorts/cohorts-page.png)
 
```

Comment:
Set your filters for the cohort and press save?

Reply:
Not necessarily, since you either need to upload a csv, or set your filters. THis is explained in more detail below

I think this is sufficient here